compiler/il/OMRNode.cpp

```diff
@@ -8642,3 +8642,55 @@ OMR::Node::resetFlagsForCodeMotion()
 /**
  * Node flags functions end
  */
+
+/**
+ * \brief
+ *    Return true of the node is a load of static final field
+ */
+bool
+OMR::Node::isLoadOfStaticFinalField()
+   {
+   if (self()->getOpCode().isLoadDirect() &&
+       self()->hasSymbolReference())
+      {
+      TR::Symbol *sym = self()->getSymbol();
+      if (sym->isFinal() &&
+          sym->isStaticField())
+         return true;
+      }
+   return false;
+   }
+
+/**
+ * \brief
+ *    Return true of the node is a call with osrFearPointSymbol
+ */
```

Comment:
I'd like to see the API documentation move to the interface in the .hpp file so that downstream projects extending the classes know how to implement the specification without reading the OMR implementation.